Grok/deploy.yaml

```diff
@@ -0,0 +1,37 @@
+---
+version: "2.0"
+services:
+  service-1:
+    image: zblocker64/grok-akash:latest
+    expose:
+      - port: 8080
+        as: 80
+        to:
+          - global: true
+profiles:
+  compute:
+    service-1:
+      resources:
+        cpu:
+          units: 16
+        memory:
+          size: 700GB
+        storage:
+          - size: 2000Gi
+        gpu:
+          units: 8
+          attributes:
+            vendor:
+              nvidia:
+                - model: h100
+  placement:
```

Comment:
please add:

```
      signedBy:
        anyOf:
          - "akash1365yvmc4s7awdyj3n2sav7xfx76adc6dnmlx63"
          - "akash18qa2a2ltfyvkyj0ggj3hkvuj6twzyumuaru9s4"
```

Like in this example https://github.com/akash-network/awesome-akash/blob/master/CodiMD/deploy.yaml#L50C20-L54C59